src/sinks/blackhole.rs

```diff
@@ -25,14 +25,14 @@ pub struct BlackholeSink {
 #[serde(deny_unknown_fields, default)]
 #[derivative(Default)]
 pub struct BlackholeConfig {
-    #[derivative(Default(value = "1000"))]
-    #[serde(default = "default_print_amount")]
-    pub print_amount: usize,
+    #[derivative(Default(value = "10"))]
+    #[serde(default = "default_print_interval_secs")]
+    pub print_interval_secs: u64,
```

Comment:
What would you think of a default of `1`? I think events/s is pretty useful as a default.

Reply:
@jszwedko Makes sense to me, yeah.  Addressed in 74ec983.